.eslintrc

```diff
@@ -50,9 +54,10 @@
     "@typescript-eslint/no-type-alias": "off",
     "@typescript-eslint/strict-boolean-expressions": "off",
     "@typescript-eslint/no-magic-numbers": "off",
-    "@typescript-eslint/no-parameter-properties": "off",
+    "@typescript-eslint/parameter-properties": "off",
     "@typescript-eslint/no-use-before-define": "off",
     "@typescript-eslint/promise-function-async": "off",
+    "@typescript-eslint/lines-around-comment": "off", // Disabled in favor of parameter-properties
```

Comment:
hm, does "lines-around-comment" has anything to do with "parameter-properties"? :upside_down_face:  The comment is confusing.

Reply:
Because I copy pasted it 😂 I'll fix it, good catch again

Reply:
hmm, I looked at this and it sounded confident so I didn't question it... :thinking:

Reply:
The reason why is `@typescript-eslint` inherits from `eslint` and apply the rules to Typescript files. They suggest to disable the `eslint` counterpart to prevent incoherent errors because `@typescript-eslint` is suppose to give the same coverage to non-ts files.